src/tests/Repository.test.js

```diff
@@ -4,6 +4,8 @@ import Repository from "../components/Repository";
 import {data, match, location} from "./mocks";
 
 describe("<Repository />", () => {
+  const getUserFromJwt = jest.fn();
+  getUserFromJwt.mockReturnValueOnce({id:29888641,login:"filiptronicek"}).mockReturnValue(true);
```

Comment:
This can be removed, I decoupled this function from this component in #777

Reply:
Ok, removed them